chart/templates/_helper.tpl

```diff
@@ -15,3 +15,7 @@
 {{ .repository }}:{{ .tag }}
 {{- end -}}
 {{- end -}}
+
+{{- define "vcluster.name" -}}
+{{- printf "vc-%s" .Release.Name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
```

Comment:
Can we make this conditional so it only prepends the `vc-` prefix if it starts with a number/non-alphabetic character?

Otherwise, helm might recreate the vCluster services on an upgrade.